app/models/mms_client.rb

```diff
@@ -40,8 +40,19 @@ def captures_for_item(uuid)
 
   private
 
+  # Actually makes the request.
+  # Throws an exception if it gets an errored response
+  def make_request_for(export_type, uuid, params = {})
+    response = authed_request.get(export_url_for(export_type, uuid), params: params)
+    if response.code >= 400
+      throw RuntimeError.new("Error getting #{export_type} for UUID #{uuid}: #{response.code} #{response}")
+    else
+      response.to_s
+    end
+  end
+
   # Builds a URL like http://metadata.nypl.org/exports/mods/123-456
```

Comment:
I changed this method name to reflect the fact that it's just building a URL string.